src/services/http.service.ts

```diff
@@ -2,20 +2,24 @@ import axios, { AxiosError, AxiosResponse } from "axios";
 import Cookies from "js-cookie";
 import psl from "psl";
 
-import { apiRequestTimeout, isLoggedInCookie } from "@constants";
+import { apiRequestTimeout, isLoggedInCookie, jwtAuthBearerToken } from "@constants";
 import { getApiBaseUrl, getCookieDomain } from "@src/utilities";
 
 const apiBaseUrl = getApiBaseUrl();
 
-const createAxiosInstance = (baseAddress: string, withCredentials = false) =>
-	axios.create({
+const createAxiosInstance = (baseAddress: string, withCredentials = false) => {
+	const isWithCredentials = !jwtAuthBearerToken || withCredentials;
+
+	return axios.create({
 		baseURL: baseAddress,
 		headers: {
 			"Content-Type": "application/x-www-form-urlencoded",
+			"Authorization": `Bearer ${jwtAuthBearerToken}`,
```

Comment:
what if jwtAuthBearerToken `undefined`? Is it the correct request?

Reply:
fixed